server/checks/Approval.js

```diff
@@ -139,9 +138,12 @@ export default class Approval extends Check {
             return
           }
           // get approvals for pr
-          const opener = pull_request.user.login
+          const commits = await github.fetchPullRequestCommits(user, repo, number, token)
+          const lastCommitter = commits.length === 0 ?
+            null :
+            commits[commits.length - 1].committer.login
           const comments = await github.getComments(user, repo, number, formatDate(dbPR.last_push), token)
-          const countConfig = Object.assign({}, config.approvals, {ignore: [opener]})
+          const countConfig = Object.assign({}, config.approvals, {ignore: lastCommitter ? [lastCommitter] : []})
```

Comment:
I could imagine the different teams would want to configure it differently. I'd prefer to not count approvals from any committer. Others might prefer the current version of ignoring the pull request creator. Maybe ignore committer(s) and opener?